contrib/python/ofi_nccl/tuner/cli/wrapper.py

```diff
@@ -49,6 +49,7 @@ class NCCLProto(int, Enum):
 class TunerPlatform(str, Enum):
     P5en = "p5en.48xlarge"
     P5 = "p5.48xlarge"
+    P6 = "p6.48xlarge"
```

Comment:
From my understanding the instance type here and in the `strcmp(platform_type, "p6.48xlarge")` check should be `p6-b200.48xlarge` (https://aws.amazon.com/ec2/instance-types/p6/ shows the EC2 instance type as `p6-b200.48xlarge`). Also looking at the `/sys/devices/virtual/dmi/id/product_name` file the OFI NCCL plugin uses by default to get the `platform_type`:
```
$ cat /sys/devices/virtual/dmi/id/product_name
p6-b200.48xlarge
```